<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QDockWidget" name="lastPlots">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>&amp;last plots</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10"/>
  </widget>
  <widget class="QDockWidget" name="createPlot">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>&amp;Create Plot</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>611</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="compareParties">
         <attribute name="title">
          <string>Gegenüberstellung Parteien</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>591</width>
            <height>541</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="allPartiesLabel">
             <property name="text">
              <string>Alle Parteien verwenden</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="selectAllPartysCheckBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="zuVerwendendeParteienAuswHlenLabel">
             <property name="text">
              <string>Zu verwendende Parteien auswählen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="diagrammtypAuswHlenLabel">
             <property name="text">
              <string>Diagrammtyp auswählen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTabWidget" name="diagrammtypAuswHlenTabWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="pieChart">
              <attribute name="title">
               <string>Kreisdiagramm</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="barchart">
              <attribute name="title">
               <string>Säulendiagramm</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="bargraph">
              <attribute name="title">
               <string>Balkendiagramm</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="anzuzeigendeDatenAuswHlenLabel">
             <property name="text">
              <string>Anzuzeigende Daten auswählen</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="anzuzeigendeDatenAuswHlenComboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QListWidget" name="partyList"/>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>550</y>
            <width>88</width>
            <height>34</height>
           </rect>
          </property>
          <property name="text">
           <string>Anzeigen</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="compareCandidates">
         <attribute name="title">
          <string>Gegenüberstellung Kandidaten</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>590</width>
            <height>491</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="allCandidatesLabel">
             <property name="text">
              <string>Alle Kandidaten verwenden</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="selectAllCandidatesCheckBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>zugehörige Partei anzeigen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="showAssociatedParty">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="zuVerwendendeKandidatenAuswHlenLabel">
             <property name="text">
              <string>Zu verwendende Kandidaten auswählen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QListWidget" name="candidatesList"/>
           </item>
           <item row="3" column="1">
            <widget class="QTabWidget" name="diagrammtypAuswHlenTabWidget_2">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="pieChart_2">
              <attribute name="title">
               <string>Kreisdiagramm</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="barchart_2">
              <attribute name="title">
               <string>Säulendiagramm</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="bargraph_2">
              <attribute name="title">
               <string>Balkendiagramm</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="anzuzeigendeDatenAuswHlenLabel_2">
             <property name="text">
              <string>Anzuzeigende Daten auswählen</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="anzuzeigendeDatenAuswHlenComboBox_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="diagrammtypAuswHlenLabel_2">
             <property name="text">
              <string>Diagrammtyp auswählen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>500</y>
            <width>88</width>
            <height>34</height>
           </rect>
          </property>
          <property name="text">
           <string>Anzeigen</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="compareElections">
         <attribute name="title">
          <string>Vergleich Wahlen</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>0</y>
       <width>861</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wählen Sie eine Vorlage aus.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Da&amp;tei</string>
    </property>
    <addaction name="actionBeenden"/>
    <addaction name="actionDatabaseSettings"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <action name="actionBeenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="actionDatabaseSettings">
   <property name="text">
    <string>&amp;settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
